### PR TITLE
OSDOCS-8467: hardcoded 4.15 version number bump

### PR DESCRIPTION
**Pull request by sheriff-rh**
Version(s):
4.15+

Issue:
https://issues.redhat.com/browse/OSDOCS-8467

Link to docs preview (VPN required):
https://file.rdu.redhat.com/antaylor/OSDOCS-8467/welcome/

Additional information:
Based on my [hardcoded 4.14 version number bump](https://github.com/openshift/openshift-docs/pull/60977) with additional updates.

### CODE REVIEW

File: machine_management/control_plane_machine_management/cpmso-troubleshooting.adoc [hunk 1]
```diff
@@ -33,7 +33,7 @@ include::modules/cpmso-ts-mhc-etcd-degraded.adoc[leveloffset=+1]
 [id="cpmso-troubleshooting-shiftstack-upgrade_{context}"]
 == Upgrading clusters that run on {rh-openstack}
 
-For clusters that run on {rh-openstack-first} that you upgrade from {product-title} 4.13 to 4.14, you might have to perform post-upgrade tasks before you can use control plane machine sets.
+For clusters that run on {rh-openstack-first} that you upgrade from {product-title} 4.14 to 4.15, you might have to perform post-upgrade tasks before you can use control plane machine sets.
```
Comment (jeana-redhat) [inline machine_management/control_plane_machine_management/cpmso-troubleshooting.adoc:36]: I will check in with dev on this, but this should probably not be changed (or should be changed in a different way). It's likely that 4.13 here is the relevant bit.

File: modules/cpmso-openstack-ts-root-volume-azs.adoc
Comment (jeana-redhat) [inline modules/cpmso-openstack-ts-root-volume-azs.adoc:1]: related to https://github.com/openshift/openshift-docs/pull/70177/files#r1452423612, hold on this one please

File: modules/cpmso-openstack-with-az-config.adoc
Comment (jeana-redhat) [inline modules/cpmso-openstack-with-az-config.adoc:1]: related to https://github.com/openshift/openshift-docs/pull/70177/files#r1452423612, hold on this one please

**Review by jeana-redhat — COMMENTED** [review]
A couple things related to my other comment, I'll chase this down today

**Review by jeana-redhat — COMMENTED** [review]
Made some suggestions based on my understanding, but this needs to be run by a ShiftStack SME. I'll direct someone here shortly :)

**Comment by mdbooth** [comment]
@EmilienM Can you take a look at the above query for me? I think there's always the possibility of having to do manual tasks before enabling CPMS on an upgraded cluster, right? The reason being we can't guarantee that the CPMS can be generated in all cases. I haven't read the full context of the comment, though, so I may be way off the mark.

**Comment by jeana-redhat** [comment]
> @EmilienM Can you take a look at the above query for me? I think there's always the possibility of having to do manual tasks before enabling CPMS on an upgraded cluster, right? The reason being we can't guarantee that the CPMS can be generated in all cases. I haven't read the full context of the comment, though, so I may be way off the mark.

If it's helpful to see the rendered output on this one, this is where that content appears in 4.14 docs: https://docs.openshift.com/container-platform/4.14/machine_management/control_plane_machine_management/cpmso-troubleshooting.html#cpmso-troubleshooting-shiftstack-upgrade_cpmso-troubleshooting

**Comment by EmilienM** [comment]
> I think there's always the possibility of having to do manual tasks before enabling CPMS on an upgraded cluster, right?

Yes, that's what we actually recommend:

* https://access.redhat.com/solutions/7013893
* https://access.redhat.com/solutions/7024383

The user HAS to manually change their cluster before CPMS can be enabled.

**Comment by jeana-redhat** [comment]
> > I think there's always the possibility of having to do manual tasks before enabling CPMS on an upgraded cluster, right?
> 
> Yes, that's what we actually recommend:
> 
>     * https://access.redhat.com/solutions/7013893
> 
>     * https://access.redhat.com/solutions/7024383
> 
> 
> The user HAS to manually change their cluster before CPMS can be enabled.

Do you think he suggestions I have made [here](https://github.com/openshift/openshift-docs/pull/70177#pullrequestreview-1822164030) accurately represent the requirements?

File: virt/support/virt-collecting-virt-data.adoc [hunk 2]
```diff
@@ -34,7 +34,7 @@ Collecting data about your environment minimizes the time required to analyze an
 .Procedure
 
 . xref:../../support/gathering-cluster-data.adoc#support_gathering_data_gathering-cluster-data[Collect must-gather data for the cluster].
-. link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.14/html-single/troubleshooting_openshift_data_foundation/index#downloading-log-files-and-diagnostic-information_rhodf[Collect must-gather data for {rh-storage-first}], if necessary.
+. link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.15/html-single/troubleshooting_openshift_data_foundation/index#downloading-log-files-and-diagnostic-information_rhodf[Collect must-gather data for {rh-storage-first}], if necessary.
```
Comment (ousleyp) [inline virt/support/virt-collecting-virt-data.adoc:37]: LGTM from the CNV perspective :)

**Pull request merged**